samples/dnn/scene_text_recognition.cpp

```diff
@@ -52,7 +52,7 @@ int main(int argc, char** argv)
     // Load vocabulary
     CV_Assert(!vocPath.empty());
     std::ifstream vocFile;
-    vocFile.open(vocPath);
+    vocFile.open(samples::findFile(vocPath));
```

Comment:
I wonder whether I need to add `samples::findFile()` whenever I open files?
Do I need to change `TextRecognitionModel recognizer(modelPath)` into `TextRecognitionModel recognizer(samples::findFile(modelPath))` ?

Reply:
`samples::findFile()` helps to use file by name (instead of full path) from `<opencv>/samples/data` location.

Model file is not stored there (and no plans to put it there due its size).